UnityProject/Assets/Scripts/UI/Core/ChangeLogs/ChangeLogHandler.cs

```diff
@@ -11,23 +19,61 @@ public class ChangeLogHandler : MonoBehaviour
 		public Transform content;
 		private List<ChangeLogEntry> allEntries = new List<ChangeLogEntry>();
 		public GameObject entryPrefab;
+		public string ChangeLogLink = "https://changelog.unitystation.org/all-changes?format=json&limit=50";
+		public string ChangeLogParams = "all-changes?format=json&limit=50";
 
 		void Start()
 		{
-			window.SetActive(false);
+			if (string.IsNullOrEmpty(ChangeLogLink))
+			{
+				window.SetActive(false);
+				return;
+			}
+			GrabJsonData();
+		}
+
+		[Button("Get Data")]
+		private void GrabJsonData()
+		{
+			var client = new HttpClient();
+			client.BaseAddress = new Uri(ChangeLogLink);
+			// Add an Accept header for JSON format.
+			client.DefaultRequestHeaders.Accept.Add(
+				new MediaTypeWithQualityHeaderValue("application/json"));
+			// Get data response
+			var response = client.GetAsync(ChangeLogParams).Result;
+			response.EnsureSuccessStatusCode();
+
+			//if we failed getting the data
+			if (response.IsSuccessStatusCode == false)
```

Comment:
This if statement will never be used, as `EnsureSuccessStatusCode` throws an exception.